content/miq_dialogs/miq_provision_ibm_dialogs_template.yaml

```diff
@@ -13,9 +13,46 @@
       :display: :hide
 
     :customize:
-      :fields: {}
-      :description: Extra
-      :display: :hide
+      :description: Customization
+      :fields:
+        :ip_addr:
+          :description: IP v4 Address
```

Comment:
If possible, we should try and match these descriptions to what is in the web UI:
"Specify an IPv4 address"

![image](https://user-images.githubusercontent.com/1637291/92963185-ad887000-f437-11ea-89ab-9f629210da0d.png)